### DISCUSSION TITLE
Descriptions are flipped for "deferred" and "recurring" in [@types/applepayjs]

### DISCUSSION THREAD

**Discussion by mintkey** [DefinitelyTyped/DefinitelyTyped#69100] [Issues with a @types package]
Lines 386-374 in `index.d.ts` have flipped descriptions for "deferred" and "recurring" payment timings. Not a big deal, but threw me off as I was going through the file.

[a comment by an automated account is not shown]

**Reply by martincostello** [reply]
Likely caused by #68660 - would be trivial to open a PR to swap them to be in the right order.

**Comment by MysteryBlokHed** [comment]
I just opened #69105
